docker/Dockerfile.dev

```
@@ -0,0 +1,17 @@
+# To build this container, go to ESMValTool root folder and execute:
+# docker build -t ${TAG_OF_YOUR_CHOICE} . -f docker/Dockerfile.dev
+FROM continuumio/miniconda3
+
+COPY . /src/ESMValTool
+WORKDIR /src/ESMValTool
+
+RUN conda update -y conda pip && conda env update --name base --file environment.yml && conda clean --all -y
+RUN pip install . && pip cache purge
```

```suggestion
RUN pip install .[test] && pip cache purge
```
to also include the test dependencies?